### PR TITLE
Add MacStroke.app v2.0.1

### PR DESCRIPTION
**Pull request by mtjo**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/macstroke.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "macstroke" do
+  version "2.0.1"
+  sha256 "c577aec2791f33a95ce0a33fe9e3691ae91c64d3bd02cc71dde6a24e1f71d7de"
+
+  url "https://github.com/mtjo/MacStroke/releases/download/#{version}/MacStroke.zip"
+  name "macstroke"
+  desc "MacStroke is a Configurable global mouse gesture for macOS.
+  MacStroke 是 macOS上一款高度可配置的全局鼠标手势软件。"
```
Comment (SMillerDev) [inline Casks/macstroke.rb:8]: I think this should be something like this? @Homebrew/cask ?
https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/firefox.rb#L20-L23

File: Casks/macstroke.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask "macstroke" do
+  version "2.0.1"
+  sha256 "c577aec2791f33a95ce0a33fe9e3691ae91c64d3bd02cc71dde6a24e1f71d7de"
+
+  url "https://github.com/mtjo/MacStroke/releases/download/#{version}/MacStroke.zip"
+  name "Macstroke"
+  desc "Make new mouse gestures"
+  homepage "https://github.com/mtjo/MacStroke"
+
+  app "MacStroke.app"
```
Comment (vitorgalvao) [inline Casks/macstroke.rb:10]: Please add a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md). See [`homebrew-createzap`](https://github.com/nrlquaker/homebrew-createzap) for an external project which can help.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/macstroke.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/macstroke.rb]: It was a continuation of the description. It was multi-line.

Comment (SMillerDev) [reply Casks/macstroke.rb]: I saw that, but what if people want a description in multiple languages?

Comment (vitorgalvao) [reply Casks/macstroke.rb]: We don’t have a precedent for that. And I’ll argue that having a `language` block just for a description complicates the cask for very little gain.

It’d make more sense to allow multiple `desc` stanzas, like we allow multiple `name` stanzas which can cover multiple languages.

**Pull request closed without merging**